test/unit/utils.js

```diff
@@ -28,6 +28,7 @@ describe('transformSdp', () => {
         "a=rtcp-mux\r\n" +
         "a=rtpmap:111 opus/48000/2\r\n" +
         "a=rtcp-fb:111 transport-cc\r\n" +
+        "a=rtcp-fb:* transport-cc\r\n" +
```

Comment:
I see that you are updating the existing unit-test -
 1. Do you need a new unit-test to cover this path? 
 2. Are the previous all the previous paths being tested even with the addition of this line?

Reply:
I think I do not need a unit-test. the code coverage should be the same if we add a new test with no `a=rtcp-fb:*`

The unit tests here makes sure `a=rtcp-fb:number` is treated properly according to the `forceCodec` codec. With the addition of this line, `a=rtcp-fb:111` is still get tested properly that in 'Removes nothing without matching media type' it is not removed, and in 'Override audio codec to pcmu', it is removed.